crud/select/executor.lua

```diff
@@ -83,6 +83,16 @@ function executor.execute(space, index, filter_func, opts)
         end
     end
 
+    if opts.after_tuple ~= nil then
+        if opts.tarantool_iter == box.index.LE then
+            opts.tarantool_iter = box.index.LT
+        elseif opts.tarantool_iter == box.index.EQ then
+            opts.tarantool_iter = box.index.GE
+        elseif opts.tarantool_iter == box.index.REQ then
+            opts.tarantool_iter = box.index.LT
+        end
+    end
```

Comment:
I think it could be moved to the plan

Reply:
I was sure that we already have such code in the plan...

Reply:
> I was sure that we already have such code in the plan...

We have only pain